Scripts/phylotol.py

```diff
@@ -208,6 +190,14 @@ def main():
 	writelog(PathtoOutput,'colcutoff = ' + str(colcutoff))
 	writelog(PathtoOutput,'rescutoff = ' + str(rescutoff))
 	writelog(PathtoOutput,'concatAlignment = ' + concatAlignment + ' (y = remove paralogs and make alignment, n = keep paralogs and do not make alignment)')
+	writelog(PathtoOutput,'majorClades = ' + str(majorClades))
+	
+# MACR -- V4 -- Added this method here for cleaning intermediary files and logs (for instance, after incomplete run or forced stoppage) using phylotol 	
+	if clean == "y": 
+		Utilities.cleaner(testPipelineList, PathtoFiles, PathtoOutput)
+		print("cleaning folders -- done!")
+		quit()	
+	
```

Comment:
MACR - This is new. It allows the user to use the method...

python phylotol.py cl

cleaning temporary files